Email.md

```diff
@@ -1,14 +1,14 @@
 Hi all,
 
-This only really pertains to programming, so if your not intrested in that you can ignore this. Also, I’m gonna try and send this to some of the 8th graders we talked to a few weeks ago as well, so I've included some more formal introductions. 
+This only really only pertains to programming, so if you're not interested in that you can ignore this. Also, I’m going to try and send this to some of the 8th graders we talked to a few weeks ago as well, so I've included some more formal introductions. 
 
-My name is George, and along with Aiden, we are in charge of programming for the high school robotics team. Last week some of the 8th graders asked for some resources to get started programming in C++, which will be used on the robot next year, but I didn’t have anything. So as not to leave them empty-handed, I thought I'd make a "little” list. To be clear, this isn’t a “required” summer assignment.
+My name is George Jurgiel, and along with Aiden Lambert, am in charge of programming for the high school robotics team. Last week some of the 8th graders asked for some resources to get started programming in C++, which will be used on the robot next year, but I didn’t have anything. So, as not to leave them empty-handed, I thought I'd make a "little” list. To be clear, this isn’t a “required” summer assignment.
```

Comment:
You should change "last week" to something else. It wasn't last week was it? Time flies.

Otherwise, I think the additions you made as well as splitting the repo into two .md files made it much more readable.

:+1: